noether_tpp/include/noether_tpp/tool_path_modifiers/blending_modifiers.h

```diff
@@ -0,0 +1,58 @@
+﻿#pragma once
+
+#include <noether_tpp/core/tool_path_modifier.h>
+
+namespace noether
+{
+/**
+ * @brief Transforms the waypoints to correspond with the center of the ginding tool so that the edge
+ * of the tool is in contact with the media
+*/
+class AngledOrientationModifier : public ToolPathModifier
+{
+public:
+  AngledOrientationModifier(double angle_offset, double tool_radius, bool flip_sign);
+  ToolPaths modify(ToolPaths tool_paths) const override final;
+
+protected:
+  const double angle_offset_;
+  const double tool_radius_;
+  const bool flip_sign_;
+};
+
+/**
+ * @brief Modifier that adjusts the parameters of the tool approach trajectory to the media
+ * @details Asumes that the x direction of the waypoints is the direction of travel for the tool.
+ * If this is not a valid assumption, run the DirectionOfTravel orientation modifier before this modifier.
+*/
+class LeadInModifier : public ToolPathModifier
```

Comment:
Let's rename this and the associated GUI classes to something like `CircularLeadInToolPathModifier` since there are other types of lead-in approaches we could make in the future